docs/website/docs/dlt-ecosystem/verified-sources/sql_database.md

```diff
@@ -438,211 +557,120 @@ source = sql_database(
-
-   ```py
-   import hashlib
-
-   def pseudonymize_name(doc):
-      '''
-      Pseudonmyisation is a deterministic type of PII-obscuring
-      Its role is to allow identifying users by their hash,
-      without revealing the underlying info.
-      '''
-      # add a constant salt to generate
-      salt = 'WI@N57%zZrmk#88c'
-      salted_string = doc['rfam_acc'] + salt
-      sh = hashlib.sha256()
-      sh.update(salted_string.encode())
-      hashed_string = sh.digest().hex()
-      doc['rfam_acc'] = hashed_string
-      return doc
-
-   pipeline = dlt.pipeline(
-       # Configure the pipeline
-   )
-   # using sql_database source to load family table and pseudonymize the column "rfam_acc"
-   source = sql_database().with_resources("family")
-   # modify this source instance's resource
-   source = source.family.add_map(pseudonymize_name)
-   # Run the pipeline. For a large db this may take a while
-   info = pipeline.run(source, write_disposition="replace")
-   print(info)
-   ```
-
-1. To exclude columns, such as the "rfam_id" column from the "family" table before loading:
-
-   ```py
-   def remove_columns(doc):
-       del doc["rfam_id"]
-       return doc
-
-   pipeline = dlt.pipeline(
-       # Configure the pipeline
-   )
-   # using sql_database source to load family table and remove the column "rfam_id"
-   source = sql_database().with_resources("family")
-   # modify this source instance's resource
-   source = source.family.add_map(remove_columns)
-   # Run the pipeline. For a large db this may take a while
-   info = pipeline.run(source, write_disposition="replace")
-   print(info)
-   ```
+### Running on Airflow
+When running on Airflow:
+1. Use the `dlt` [Airflow Helper](../../walkthroughs/deploy-a-pipeline/deploy-with-airflow-composer.md#2-modify-dag-file) to create tasks from the `sql_database` source. You should be able to run table extraction in parallel with `parallel-isolated` source->DAG conversion.
+2. Reflect tables at runtime with `defer_table_reflect` argument.
+3. Set `allow_external_schedulers` to load data using [Airflow intervals](../../general-usage/incremental-loading.md#using-airflow-schedule-for-backfill-and-incremental-loading).
 
-1. Remember to keep the pipeline name and destination dataset name consistent. The pipeline name is crucial for retrieving the [state](https://dlthub.com/docs/general-usage/state) from the last run, which is essential for incremental loading. Altering these names could initiate a "[full_refresh](https://dlthub.com/docs/general-usage/pipeline#do-experiments-with-full-refresh)", interfering with the metadata tracking necessary for [incremental loads](https://dlthub.com/docs/general-usage/incremental-loading).
+### Transforming the data before load
+You have direct access to the extracted data through the resource objects (`sql_table()` or `sql_database().with_resource())`), each of which represents a single SQL table. These objects are generators that yield 
```

Comment:
Please add a note somewhere, that with pyarrow BE, you need to adjus function you pass to `add_map` to work with pyarrow chunks. Reference: https://dlthub-community.slack.com/archives/C04DQA7JJN6/p1721686911160439